tools/run_make_firmwares3.py

```diff
@@ -30,7 +30,7 @@ def findSTM32CubeIDEGnuTools(search_root):
     st_cubeide_dir = ''
     st_cubeide_ver_nr = 0
     for file in os.listdir(search_root):
-        if 'STM32CubeIDE' in file:
+        if 'stm32cubeide' in file or 'STM32CubeIDE' in file:
```

Comment:
maybe better something line `if 'stm32cubeide' in file.lower(): # to make it work on several os`